src/tiledimage.js

```diff
@@ -770,18 +773,17 @@ $.extend($.TiledImage.prototype, $.EventSource.prototype, /** @lends OpenSeadrag
     },
 
     /**
-     * @returns {Boolean} whether the tiledImage is set to load tiles without drawing them.
+     * @returns {Boolean} whether the tiledImage can load hidden tiles of zero opacity.
```

Comment:
The wording on this comment could probably be better... at the moment it sounds like some certain tiles might be hidden or have 0 opacity... it's the whole image that has 0 opacity. Something like "whether the tiledImage can load its tiles even when it has zero opacity."